crates/scheduler/tests/worker_pool_tests.rs

```diff
@@ -0,0 +1,178 @@
+use crossbeam::channel::unbounded;
+use crossbeam::sync::Parker;
+//noinspection RsUnusedImport - CLion can't recognize that `Write` is being used.
```

Comment:
My CLion installation had no problem detecting that Write and timeout were being used when I removed noinspection.

Reply:
Linux supremacy? 😩 

or, it might be because I've got a bunch of preview features enabled to make it better analyze procedural macros.